hopp/tools/design/wind/power_curve_tools.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import copy
+
+def plot_power_curve(wind_speeds_ms, cp_curve, ct_curve):
+    """Plot Cp and Ct curve per wind speed.
+
+    Args:
+        wind_speeds_ms (List | np.ndarray): list of wind speeds in m/s
+        cp_curve (List | np.ndarray): power curve coefficients (Cp) at each wind speed in `wind_speeds_ms`
+        ct_curve (_type_): thrust curve coefficients (Ct) at each wind speed in `wind_speeds_ms`
```

Comment:
`List` -> `list` and `ct_curve` should be of the same argument type as the others.

Reply:
There are a handful of spots where `List` is used for the docstring typing in this file, can you replace those as well?